dali/base/dautils.hpp

```diff
@@ -577,4 +577,10 @@ inline unsigned calcStripeNumber(unsigned partNum, const char *lfnName, unsigned
 interface INamedGroupStore;
 extern da_decl void remapGroupsToDafilesrv(IPropertyTree *file, INamedGroupStore *resolver);
 
+#ifdef NULL_DALIUSER_STACKTRACE
+extern da_decl void logNullUser(IUserDescriptor *userDesc);
```

Comment:
Note: This function won't be available outside dali - because NULL_DALIUSER_STACKTRACE will not be defined when it is included.  Not actually a problem, but there is no benefit in the da_decl which exports the symbol.